modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/hooks/useValidateFormWithObjects.js

```diff
@@ -75,7 +75,10 @@ export function useValidateFormWithObjects() {
 				}
 			});
 
-			if (hasObjectDefinitionId && isStorageTypeObject) {
+			if (
+				(hasObjectDefinitionId && isStorageTypeObject) ||
+				!!objectFields[0]
```

Comment:
Hey @manuelecastro, you can just use `objectFields.length` to make this condition